setup.py

```diff
@@ -73,10 +73,13 @@
         'notebook',
         's3fs',
         'tornado >= 5.1.1',
-        'aiobotocore>=0.10.0',
+        'aiobotocore==0.10.2',
```

Comment:
Any reason we don't read this from the requirements.txt? This requirement is different here and I have found that having duplicate dependency source files causes this problem often.

Reply:
Oversight that they diverged. Do we want requirements.txt or only in setup.py?

Reply:
Personally I like the `requirement.txt` as a source of truth. It's clean and simple for someone to find the requirements. I've done this in papermill and scrapbook as well. Doesn't have to be done in this PR, and so long as we collapse to one place it should be fine.

Reply:
Sounds good. Looks like bumping to Sphinx 2+ and unpinning resolved. I have not idea why but 🤷‍♀

Reply:
@MSeal spoke too soon. Needed to wipe and restart the rtd container. Darn dependencies.